oozie/src/main/java/org/apache/falcon/oozie/process/SparkProcessWorkflowBuilder.java

```diff
@@ -163,13 +164,16 @@ private void addOutputFeedsAsArgument(List<String> argList, Cluster cluster) thr
             return;
         }
 
-        for(Output output : entity.getOutputs().getOutputs()) {
+        int numOutputFeed = entity.getOutputs().getOutputs().size();
```

Comment:
Can you add a comment on this (that we are adding to the 0th index and getting the args shifted as arguments are added to get the desired effect).